lib/LaTeXML/resources/XSLT/LaTeXML-block-xhtml.xsl

```diff
@@ -155,7 +155,9 @@
           <xsl:with-param name="name" select="'href'"/>
         </xsl:call-template>
         <xsl:if test="$USE_HTML5='true'">
-          <xsl:attribute name="download"/>
+          <xsl:attribute name="download">
+            <xsl:value-of select="@name"/>
```

Comment:
This wants to be `@dataname`, right?

Reply:
...forgot to run the final `git commit --amend`...